### ISSUE TITLE
Bad Gateway 502 with traefik

### ISSUE THREAD

**Issue by YoMamasNerd** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#471]
**Describe the bug**
I'm using traefik to expose the web analytics to the web and it shows a 502 bad gateway.

**To Reproduce**
1. Set up a working Docker setup
2. install a working traefik instance
3. install twitch miner
4. try to expose the analytics

**Expected behavior**
The analytics should show up at the expected URL

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Docker host
 - newest Docker image

**Additional context**
My docker config: [(Image for better readability)](https://pasteboard.co/OiQPS2sUKGZV.png)
```
  twitch-miner_app:
    image: tkdalex/twitch-channel-points-miner-v2
    container_name: twitch-miner_app
    restart: unless-stopped
    tty: true
    environment:
      - TERM=xterm-256color
    volumes:
      - ./twitch-miner/analytics:/usr/src/app/analytics
      - ./twitch-miner/cookies:/usr/src/app/cookies
      - ./twitch-miner/logs:/usr/src/app/logs
      - ./twitch-miner/run.py:/usr/src/app/run.py:ro
    labels:
      - "traefik.enable=true"
      - "traefik.http.middlewares.twitch-miner-https.redirectscheme.scheme=https"
      - "traefik.http.routers.twitch-miner-http.entrypoints=web"
      - "traefik.http.routers.twitch-miner-http.rule=Host(`twitch.xxx.xxx`)"
      - "traefik.http.routers.twitch-miner-http.middlewares=twitch-miner-https@docker"
      - "traefik.http.routers.twitch-miner.entrypoints=web-secure"
      - "traefik.http.routers.twitch-miner.rule=Host(`twitch.xxx.xxx`)"
      - "traefik.http.routers.twitch-miner.tls=true"
      - "traefik.http.routers.twitch-miner.tls.certresolver=default"
      - "traefik.http.services.twitch-miner.loadbalancer.server.port=8080"
      - "com.centurylinklabs.watchtower.enable=true"
```

My run.py analytics section
```
twitch_miner.analytics(host="127.0.0.1", port=8080, refresh=5)
```

**Comment by ColinShark** [comment]
Either indent your pasted config file by four spaces or wrap it into triple backticks (<code>```</code>) to format as a code block:
<pre>```
code
here
```</pre>
<details><summary>I've taken the liberty to format this for you.</summary>

```yaml
twitch-miner_app:
  image: tkdalex/twitch-channel-points-miner-v2
  container_name: twitch-miner_app
  restart: unless-stopped
  tty: true
  environment:
    - TERM=xterm-256color
  volumes:
    - ./twitch-miner/analytics:/user/src/app/analytics
    - ./twitch-miner/cookies:/usr/src/app/cookies
    - ./twitch-miner/logs:/usr/src/app/logs
    - ./twich-miner/run.py:/usr/src/app/run.py:ro
  labels:
    - "traefik.enable=true"
    - "traefik.http.middlewares.twitch-miner-https.redirectscheme.scheme=https"
    - "traefik.http.routers.twitch-miner-http.entrypoints=web"
    - "traefik.http.routers.twitch-miner-http.rule=Host(`twitch.xxx.xxx`)"
    - "traefik.http.routers.twitch-miner-http.middlewares=twitch-miner-https@docker"
    - "traefik.http.routers.twitch-miner.entrypoints=web-secure"
    - "traefik.http.routers.twitch-miner.rule=Host(`twitch.xxx.xxx`)"
    - "traefik.http.routers.twitch-miner.tls=true"
    - "traefik.http.routers.twitch-miner.tls.certresolver=default"
    - "traefik.http.services.twitch-miner.loadbalancer.server.port=8080"
    - "com.centurylinklabs.watchtower.enable=true"
```
</details>

I do not have a solution for your problem, I have no experience with both Docker and "traefik". I'm only commenting because badly formatted code is a pet peeve of mine.

**Comment by Mouhwahno** [comment]
You should bind your the analytics to 0.0.0.0 instead of 127.0.0.1. Everything works fine on my end.

Here are my traefik labels
```
- "traefik.enable=true"
- "traefik.http.services.ttvminer.loadbalancer.server.port=5000"

- "traefik.http.routers.ttvminer.service=ttvminer@docker"
- "traefik.http.routers.ttvminer.entrypoints=webs"
- "traefik.http.routers.ttvminer.rule=Host(`ttvminer.domain.tld`)"
- "traefik.http.routers.ttvminer.tls=true"
- "traefik.http.routers.ttvminer.tls.certresolver=cert"
```
And my analytics section
```
twitch_miner.analytics(host="0.0.0.0", port=5000, refresh=15, days_ago=30)
```

**Comment by YoMamasNerd** [comment]
Thank you that fixed it..
I just spun up another container last week with the same problem. And I fixed it with setting the listen IP to 0.0.0.0.
I don't know why I didn't thought about that.
